deluge/scripts/create_plugin.py

```diff
@@ -238,7 +238,7 @@ def get_resource(filename):
 
 class GtkUI(GtkPluginBase):
     def enable(self):
-        self.glade = gtk.glade.XML(get_resource("config.glade"))
+        self.glade = Gtk.glade.XML(get_resource("config.ui"))
```

Comment:
This should be builder code like the other plugins:
```
self.main_builder = Gtk.Builder()
self.main_builder.add_from_file(get_resource("config.ui"))
```

This likely will not be the only change needing done in this script.